src/tests/ftest/daos_test/daos_core_test-rebuild.yaml

```diff
@@ -52,78 +56,61 @@ dmg:
-      args: -s3 -u subtests="18"
-    test_rebuild_19:
-      daos_test: r
-      test_name: DAOS_Rebuild_19
-      args: -s3 -u subtests="19"
-    test_rebuild_20:
-      daos_test: r
-      test_name: DAOS_Rebuild_20
-      args: -s3 -u subtests="20"
-    test_rebuild_21:
-      daos_test: r
-      test_name: DAOS_Rebuild_21
-      args: -s3 -u subtests="21"
-    test_rebuild_22:
-      daos_test: r
-      test_name: DAOS_Rebuild_22
-      args: -s3 -u subtests="22"
-      stopped_ranks: [7]
-    test_rebuild_23:
-      daos_test: r
-      test_name: DAOS_Rebuild_23
-      args: -s3 -u subtests="23"
-      stopped_ranks: [7]
-    test_rebuild_24:
-      daos_test: r
-      test_name: DAOS_Rebuild_24
-      args: -s3 -u subtests="24"
-      stopped_ranks: [7]
-    test_rebuild_25:
-      daos_test: r
-      test_name: DAOS_Rebuild_25
-      args: -s3 -u subtests="25"
-      stopped_ranks: [5, 6, 7]
-    test_rebuild_26:
-      daos_test: r
-      test_name: DAOS_Rebuild_26
-      args: -s3 -u subtests="26"
-      stopped_ranks: ["random"]
-    test_rebuild_27:
-      daos_test: r
-      test_name: DAOS_Rebuild_27
-      args: -s6 -u subtests="27"
-      test_timeout: 1500
-      stopped_ranks: ["random"]
-    test_rebuild_28:
-      daos_test: r
-      test_name: DAOS_Rebuild_28
-      args: -s3 -u subtests="28"
-      stopped_ranks: [7]
+  num_clients: 1
+  num_replicas: 1
+  test_name:
+    test_rebuild_0to10: DAOS_Rebuild_0to10
+    test_rebuild_12to15: DAOS_Rebuild_12to15
+    test_rebuild_16: DAOS_Rebuild_16
+    test_rebuild_17: DAOS_Rebuild_17
+    test_rebuild_18: DAOS_Rebuild_18
+    test_rebuild_19: DAOS_Rebuild_19
+    test_rebuild_20: DAOS_Rebuild_20
+    test_rebuild_21: DAOS_Rebuild_21
+    test_rebuild_22: DAOS_Rebuild_22
+    test_rebuild_23: DAOS_Rebuild_23
+    test_rebuild_24: DAOS_Rebuild_24
+    test_rebuild_25: DAOS_Rebuild_25
+    test_rebuild_26: DAOS_Rebuild_26
+    test_rebuild_27: DAOS_Rebuild_27
+    test_rebuild_28: DAOS_Rebuild_28
+  daos_test:
+    test_rebuild_0to10: r
+    test_rebuild_12to15: r
+    test_rebuild_16: r
+    test_rebuild_17: r
+    test_rebuild_18: r
+    test_rebuild_19: r
+    test_rebuild_20: r
+    test_rebuild_21: r
+    test_rebuild_22: r
+    test_rebuild_23: r
+    test_rebuild_24: r
+    test_rebuild_25: r
+    test_rebuild_26: r
+    test_rebuild_27: r
+    test_rebuild_28: r
+  args:
+    test_rebuild_0to10: -s3 -u subtests="0-10"
+    test_rebuild_12to15: -s3 -u subtests="12-15"
+    test_rebuild_16: -s3 -u subtests="16"
+    test_rebuild_17: -s3 -u subtests="17"
+    test_rebuild_18: -s3 -u subtests="18"
+    test_rebuild_19: -s3 -u subtests="19"
+    test_rebuild_20: -s3 -u subtests="20"
+    test_rebuild_21: -s3 -u subtests="21"
+    test_rebuild_22: -s3 -u subtests="22"
+    test_rebuild_23: -s3 -u subtests="23"
+    test_rebuild_24: -s3 -u subtests="24"
+    test_rebuild_25: -s3 -u subtests="25"
+    test_rebuild_26: -s3 -u subtests="26"
+    test_rebuild_27: -s3 -u subtests="27"
```

Comment:
noticed this after the patch landed, test_rebuild_27 states that it requires -s6.
I was iterating on a patch that tests rebuild 27 in particular, and after merging master found this difference. At run time the avocado test passes, but it seemed to go a little more quickly without anything interesting happening. Found this in the output
2021-02-11 04:05:05,229 process          L0479 DEBUG| [stdout] need at least 6 svcs, -s6

Reply:
I filed ticket https://jira.hpdd.intel.com/browse/DAOS-6792 after confirming the issue is observed on the most recent daily_regression timed build of daos master.